all-in-one/README.md

```diff
@@ -1,6 +1,6 @@
 # All-in-One Deployment
```

Comment:
Is we're using this file we may want to pull out the Build and Testing sections into their own file that's just referenced from here.

People hitting Docker Hub probably don't care about building.

Reply:
@rgauss Would you recommend I build a separate markdown file in the same directory? Something like `all-in-one/Building-Testing.md`